embedded-io-adapters/Cargo.toml

```
@@ -0,0 +1,28 @@
+[package]
+name = "embedded-io-adapters"
+version = "0.1.0"
+edition = "2021"
+description = "Adapters between the `embedded-io` traits and other I/O traits"
+repository = "https://github.com/rust-embedded/embedded-hal"
+readme = "README.md"
+license = "MIT OR Apache-2.0"
+categories = [
+    "embedded",
+    "no-std",
+]
+
+[features]
+std = []
```

```suggestion
std = ["embedded-io/std"]
```

The std feature should enable the std feature of embedded-io

---

done!